src/views/Applications/ApplicationForm.vue

```diff
@@ -305,17 +333,40 @@ export default defineComponent({
     const { portalApiV2 } = usePortalApi()
 
     onMounted(async () => {
+      const promises = []
       if (id.value) {
-        fetchApplication()
+        promises.push(fetchApplication())
+      } else {
+        // placeholder to make destructuring result work
+        promises.push('_')
```

Comment:
How come destructuring doesn't work without this?

Reply:
cause we grab only second value on line 350.